Neos.ContentRepository.LegacyNodeMigration/Classes/Command/ContentRepositoryMigrateCommandController.php

```diff
@@ -129,7 +129,8 @@ public function runCommand(bool $verbose = false, string $config = null): void
             $this->outputLine('Cancelled...');
             $this->quit();
         }
-        $this->connection->executeStatement('TRUNCATE ' . $connection->quoteIdentifier($eventTableName));
+
+        $this->connection->executeStatement('TRUNCATE '. $eventTableName . ';');
```

Comment:
It is necessary _not_ to quote the table name? This is surprising.

Cosmetic tweak
```suggestion
        $this->connection->executeStatement('TRUNCATE ' . $eventTableName);
```